src/celeritas/global/alongstep/AlongStepNeutralAction.cu

```diff
@@ -24,6 +26,7 @@ namespace celeritas
 void AlongStepNeutralAction::execute(CoreParams const& params,
                                      CoreStateDevice& state) const
 {
+    ScopedProfiling profile_this{label()};
```

Comment:
Should we be putting these in all the individual actions, or just once in the `ActionSequence::execute`?

Reply:
I think it's better to have it in the individual actions, that way we can target exactly which kernel we want to profile. We could have both and add a range in the `ActionSequence` in case we want to profile an entire step.

Reply:
I'm not suggesting one label for the whole step; I'm saying we should be able to use an action label inside the event loop:
```diff
diff --git a/src/celeritas/global/detail/ActionSequence.cc b/src/celeritas/glob>
index 5376ebc52..c91e5110f 100644
--- a/src/celeritas/global/detail/ActionSequence.cc
+++ b/src/celeritas/global/detail/ActionSequence.cc
@@ -103,6 +103,15 @@ void ActionSequence::execute(CoreParams const& params, Cor>
             accum_time_[i] += get_time();
         }
     }
+    else if (M === MemSpace::device)
+    {
+        // Just loop over the actions
+        for (SPConstExplicit const& sp_action : actions_)
+        {
+            ScopedProfiling profile_this(sp_action->label());
+            sp_action->execute(params, state);
+        }
+    }
     else
     {
         // Just loop over the actions
```

Reply:
yes that's less repetitions, and keep multiple ranges for actions that launch multiple kernels.